specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2023-07-01/imagebuilder.json

```diff
@@ -0,0 +1,2033 @@
+    "UserAssignedIdentities": {
+      "title": "User-Assigned Identities",
+      "description": "The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/UserAssignedIdentity"
+      }
+    },
+    "UserAssignedIdentity": {
+      "type": "object",
+      "description": "User assigned identity properties",
+      "properties": {
+        "principalId": {
+          "description": "The principal ID of the assigned identity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "clientId": {
+          "description": "The client ID of the assigned identity.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RunOutputProperties": {
+      "type": "object",
+      "description": "Describes the properties of a run output",
+      "properties": {
+        "artifactId": {
+          "type": "string",
+          "description": "The resource id of the artifact."
+        },
+        "artifactUri": {
+          "type": "string",
+          "description": "The location URI of the artifact."
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "Provisioning state of the resource",
+          "readOnly": true
+        }
+      }
+    },
+    "ImageTemplate": {
+      "type": "object",
+      "description": "Image template is an ARM resource managed by Microsoft.VirtualMachineImages provider",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ImageTemplateProperties",
+          "description": "The properties of the image template"
+        },
+        "identity": {
+          "$ref": "#/definitions/ImageTemplateIdentity",
+          "description": "The identity of the image template, if configured."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "required": [
+        "identity"
+      ],
+      "x-ms-identifiers": []
+    },
+    "ImageTemplateUpdateParameters": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/ImageTemplateIdentity",
+          "description": "The identity of the image template, if configured."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The user-specified tags associated with the image template."
+        },
+        "distribute": {
```

Comment:
For the last lint diff error, the root cause is at this line.

The definition `ImageTemplateUpdateParameters` is used to update the resource. But the definition of `distribute` is at a incorrect level, it should be like

```
identity: {}
tags: {}
properties: {
      distribute: {}
}
``` 

but now, `distribute` is at the same level with `tags` and `identity`, please check and fix it.

Reply:
I pushed a new commit and it created a ton of new errors. I'm online now for the next hour if you are available to help me fix this. There is some urgency to this PR as I have deadlines to meet. Thank you